pkg/comp-functions/functions/vshnpostgres/restart.go

```diff
@@ -16,7 +16,7 @@ import (
 	controllerruntime "sigs.k8s.io/controller-runtime"
 )
 
-var sgDbOpsRestartRetention time.Duration = 30 * 24 * time.Hour
+var sgDbOpsRestartRetention time.Duration = 10 * time.Minute
```

Comment:
Can you add a comment here? why this is now much lower?

Reply:
It's actually explained in the commit message:

On clusters with many instance there can be thousands of old sgdbops restarts that noone cares about and just generate load.